cosmwasm/submit-proposal.js

```diff
@@ -331,7 +338,7 @@ const programHandler = () => {
             options.chains = chains;
             return mainProcessor(registerItsChain, options);
         });
-    addAmplifierOptions(registerItsChainCmd, { proposalOptions: true, runAs: true });
+    addAmplifierOptions(registerItsChainCmd, { proposalOptions: true, runAs: true, itsTranslatorAddress: true });
```

Comment:
why add this as an arg to the common util? It's very specific to this cmd, so add it as an option to the `registerItsChainCmd` command above?